docs/book/src/quick-start.md

```diff
@@ -12,9 +12,18 @@ This Quick Start guide will cover:
 - [go](https://golang.org/dl/) version v1.15+.
 - [docker](https://docs.docker.com/install/) version 17.03+.
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) version v1.11.3+.
-- [kustomize](https://sigs.k8s.io/kustomize/docs/INSTALL.md) v3.1.0+
 - Access to a Kubernetes v1.11.3+ cluster.
 
+<aside class="note">
+<h1>Versions and Supportability</h1>
+
+Projects created by Kubebuilder contain a Makefile that will install tools at versions defined at creation time. Those tools are:
+- [kustomize](https://kubernetes-sigs.github.io/kustomize/)
+- [controller-gen](https://github.com/kubernetes-sigs/controller-tools)
+
+Note that, the officially supported versions are that ones defined in the Makefile and go.mod files which are generated by tool. 
```

Comment:
No `go.mod` entry is created for any tool since they're installed in temp modules outside of a project tree
```suggestion
We test with and therefore recommend using the defined versions.
```

Reply:
The pronoun `we` should be avoided. See: https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we

Also, I'd like to make clear that the versions on go.mod and in the Makefile are the supported ones. I mean, users are able to customize it as they wish, however, we are unable to ensure the behaviour with other versions. I tried to address your suggestion and keep the idea/motivation. Let's see. Please, feel free to make a new suggestion if you wish as well.